app/controllers/hosts_controller.rb

```diff
@@ -48,18 +48,20 @@ def index(title = nil)
       error e.to_s
       search = resource_base.search_for ''
     end
+    @hosts = search.includes(included_associations).paginate(:page => params[:page])
```

Comment:
I think power status would require much too long for it to be useful as it will have to query external apis. Not sure about status, might also be complicated to include as it is computed currently in a complex way and not as a method on host that we can easily call. I'll remove for now and see how that goes.